Sources/Jaeger/Jaeger/Tracer/JaegerTracer.swift

```diff
@@ -39,14 +39,11 @@ public final class BasicTracer: Tracer {
      */
     public func startSpan(operationName: String, referencing reference: Span.Reference?, startTime: Date, tags: [Tag]) -> OTSpan {
 
-        let refs = [reference].compactMap { $0 }
-
         let span = Span(
             tracer: self,
             spanRef: .init(traceId: self.tracerId, spanId: UUID()),
-            parentSpanId: reference?.context.spanId,
+            parentSpan: reference,
```

Comment:
Since we are only passing in a span reference as opposed to a complete span, should we call this parameter `parentSpanReference` instead of `parentSpan `?

Reply:
Done :)